code-snippet-plugin/src/test/java/com/okta/maven/snippet/Example.java

```diff
@@ -18,12 +18,17 @@
 public class Example {
 
     public void doFoo() {
-        System.out.println("doing foo");
+        System
+          .out.println("doing foo");
```

Comment:
Was this change intended? Seems that this `System.out.println` can fit any width-rules.

Reply:
This is the input for a test, to make sure the output ends up in the same format